tests/scripts/create-dev-cluster.sh

```diff
@@ -147,41 +173,56 @@ while getopts ":hrmfd:" opt; do
 	d)
 	    ROOK_EXAMPLES_DIR="$OPTARG"
 	    ;;
+    y)
+	    minikube_driver="$OPTARG"
+	    ;;
 	\?)
-	    echo  "Invalid option: -$OPTARG" >&2
+	    echo  "Invalid option: -$opt" >&2
 	    show_usage
 	    exit 1
 	    ;;
 	:)
-	    echo "Option -$OPTARG requires an argument." >&2
+	    echo "Option -$opt requires an argument." >&2
 	    exit 1
 	    ;;
     esac
 done
 
-echo "Using '$ROOK_EXAMPLES_DIR' as examples directory.."
+echo "Using '${ROOK_EXAMPLES_DIR}' as examples directory.."
 
-cd "$ROOK_EXAMPLES_DIR" || exit
+cd "${ROOK_EXAMPLES_DIR}" || exit
 check_examples_dir
 
-if [ -z "$force_minikube" ]; then
-    check_minikube_exists
+    check_minikube_profile_exists
+    exists=$?
+    if [[ $exists -eq 0 ]]; then
+if [[ -n "${force_minikube}" ]]; then
+    if [[ ${exists} -eq 0 ]]; then
+if [[ -z "${force_minikube}" ]]; then
+    check_minikube_profile_exists
+    exists=$?
+    if [[ ${exists} -eq 0 ]]; then
+    	setup_minikube_env
+    fi
 fi
 
-setup_minikube_env
+fi
+fi
+    fi
+
 create_rook_cluster
 wait_for_rook_operator
 wait_for_ceph_cluster
 
-if [ "$enable_rook" = true ]; then
+if [[  "${enable_rook}" == "true" ]]; then
     enable_rook_orchestrator
 fi
 
-if [ "$enable_monitoring" = true ]; then
+if [[ "${enable_monitoring}" == "true" ]]; then
```

Comment:
`[[` is in general safe but it's an expansion and it's not POSIX thus less portable than simple `[`. In relation to the usage of "{}" I think  it's not needed in this case as we are not dealing with params grouping. That's said I'll keep it on something like: `if [ "$enable_monitoring" = "true" ]; then`  (the `=` is a valid operator for string comparison)

Reply:
@rkachach this is already a bash script, so why insist of keeping it POSIX and portable?

And to explain, I switched to `${name}` for variable expansion because I generally find it more readable, and I switched to `[[ ... ]]` for if statements since I kept getting syntax errors trying to run the script with `if [ ...] ` Similar for `==` vs `=`: I used what worked for me...

Reply:
`[[ ... ]]` is available for all bash platforms, but it isn't part of the unix shell requirements, so other shells might not have it.